CHARTER.md

```diff
@@ -0,0 +1,140 @@
+
+The Governing Board helps the Community to define our scope and direction, and is responsible for the organization finances.
+
+The Board is a group of 5 people who represent the MapLibre Organization.
+
+While the Governing Board’s purpose is not to handle technical topics in general, the Board retains the authority over all decisions the MapLibre Projects make, and may overrule the Technical Steering Committee in case of a dispute.
+
+No more than one person on the Governing Board shall be employed or contracted by a single third-party organization.
+
+The Board shall have at least the following roles:
+
+* Treasurer: Responsible for financial transactions and accounting.
+* Secretary: Responsible for taking and publishing meeting notes and management of Voting membership.
+
+The Governing Board meets at least 4 times per year, publishes quarterly financial reports, and shares the content of their meetings with the MapLibre Community to the extent the Board deems appropriate.
+
+### Voting Members
+
+Voting Members are people who elect the Governing Board and who vote on changes to the MapLibre Charter.
+
+A precondition to becoming a Voting Member is that the person contributed in a non-trivial way or that they donated funds to the MapLibre Organization.
+
+If Voting Members are paid by a Third-party organization for their work on the MapLibre Projects, they shall declare publicly who their employer is. At most 10 percent of the Voting Members shall be employed by a single Third-party organization.
+
+Voting Members shall inform the MapLibre Organization if their work situation with respect to being paid for working on MapLibre Projects changes.
+
+Voting Members can nominate other Community members to receive Voting membership in acknowledgement of their contributions to the MapLibre Projects. The Voting Members then vote on granting the nominee a Voting membership.
+
+Nominations for new Voting Members are open up to two weeks before the election of the Governing Board and the vote on granting Voting Membership to somebody takes place one week before the election of the Governing Board.
+
+### Governing Board Election
+
+Governing Board elections are announced at least four weeks in advance.
+
+Nomination process:
+
+* A person can nominate themselves as a candidate for the Governing Board.
+* Candidates have to register themselves up to 2 weeks before the election.
+
+Voting process:
+
+* Governing Board elections take place every year.
+* Each Voting Member ranks all nominees according to ranked-choice voting.
+* The nominee with the largest number of votes gets appointed, the nominee with the second largest number of votes gets appointed, and so on.
+* In case there are multiple nominees per Third-party organization, only the one with the largest number of votes gets appointed.
+* In case of a tie where more nominees than remaining seats receive an equal number of votes, the vote shall be repeated for the remaining seats.
```

Comment:
```suggestion
* In case of a tie where more nominees than remaining seats receive an equal ranking, the vote shall be repeated for the remaining seats.
```